dask_jobqueue/core.py

```diff
@@ -110,15 +178,23 @@ def __init__(self,
 
         self.cluster = LocalCluster(n_workers=0, ip=host, **kwargs)
 
+        # plugin for tracking job status
+        self._scheduler_plugin = JobQueuePlugin()
+        self.cluster.scheduler.add_plugin(self._scheduler_plugin)
+        self.pending_jobs = self._scheduler_plugin.pending_jobs
+        self.running_jobs = self._scheduler_plugin.running_jobs
+        self.finished_jobs = self._scheduler_plugin.finished_jobs
```

Comment:
Perhaps these should be properties?  I'm not sure exactly why I'm suggesting this, but it seems like a more common pattern.